src/app/telegram/votigram/components/VoteItem/index.tsx

```diff
@@ -176,69 +171,65 @@ export default function VoteItem(props: IVoteItemProps) {
             </div>
           </div>
         </div>
-        {showVoteAndLike &&
```

Comment:
I add this parameter to support preview on the voting page creation.

Reply:
I had fixed this